src/components/HomePage/Gallery.js

```diff
@@ -11,7 +11,7 @@ export default function Gallery() {
 					id
 					relativePath
 					childImageSharp {
-						fluid {
+						fluid(quality: 100) {
```

Comment:
Let's leave this out for https://github.com/uclaacm/hoth.uclaacm.com/issues/78 to fix.